schema/article.js

```diff
@@ -0,0 +1,43 @@
+import _ from 'lodash';
+import positron from '../lib/loaders/positron';
+import cached from './fields/cached';
+import AuthorType from './author';
+import {
+  GraphQLString,
+  GraphQLObjectType,
+  GraphQLNonNull,
+} from 'graphql';
+
+let ArticleType = new GraphQLObjectType({
+  name: 'Article',
+  fields: () => ({
+    cached: cached,
```

Comment:
I dunno if this is easily configureable in Metaphysics, but if it's helpful you have my blessing (and encouragement) to not cache responses from Positron for Positron's sake.